python/vast/vast/vast.py

```diff
@@ -76,8 +76,9 @@ def _next_batch():
 class VAST:
     """An instance of a VAST node."""
 
-    def __init__(self):
-        pass
+    def __init__(self, endpoint="127.0.0.1:42000"):
```

Comment:
agreed, if we assume that this is the equivalent to setting `--endpoint`, it should fallback to the binary configuration.  Should this maybe be a config object instead of one argument by parameter? I guess that it would make sense to add the other configs also (client binary verbosity, connection timeout...).